pkg/test/resourcefixture/testdata/basic/certificatemanager/v1beta1/certificatemanagerdnsauthorization-full/update.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Google LLC
```

Comment:
Is Update supported for this resource? If yes, we need to modify the mutable fields or labels in update.yaml. If not then we do not need update.yaml, and we can create test context for this resource and add `SkipUpdate: true` in the test context, similar to https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/8cc82c9173f45ea06d1ee718a5aceb2540b1cfed/pkg/test/resourcefixture/contexts/alloydb_context.go#L21

Reply:
nvm, per the manual the 1st PR should have the same create and update yaml.

Reply:
This is the first PR that would have create and update the same. The second PR would have update.yaml changes every mutable fields; along with golden log/object change to reflect that